.docker/base/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM projecteverest/everest-chomolungma-d8
+LABEL maintainer "Irina Spiridonova <irinasp@microsoft.com>; Darren Gehring <darrenge@microsoft.com>; Tahina Ramananandro <taramana@microsoft.com>"
+
+USER everest
+WORKDIR ${MYHOME}
+
+#PAT token should be provided in the docker run command or as a variable in VSTS build definition 
+ENV PAT $PAT
+
+#software-properties-common package is needed for add-apt-repository
+#installing latest version of git
+RUN sudo apt-get install --yes software-properties-common
+RUN sudo add-apt-repository ppa:git-core/ppa
+RUN sudo apt-get update
+RUN sudo apt-get install git nuget
+
+#Prepare everest-ci and everest-logs repositories
+RUN git clone https://$PAT@github.com/project-everest/everest-ci.git
+RUN git clone https://$PAT@github.com/project-everest/ci-logs.git
+RUN echo "export CI_LOGS=/$MYHOME/everest/ci-logs/" >> .bashrc 
+RUN . $MYHOME/.bashrc
```

Comment:
I thought every single command was evaluated in a separate sub-shell...; does this have an impact on the subsequent commands?